.github/workflows/release.yml

```diff
@@ -27,29 +32,28 @@ jobs:
           vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
           vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
       - run: npm ci # runs npm prepublish
-      - name: Bump the version
-        run: |
-          echo "VERSION=$(npm version)" >> $GITHUB_ENV
-          echo "NPM_PACKAGE=$(jq '.name' package.json) >> $GITHUB_ENV
-      - run: git push --tags
       - uses: ./.github/actions/setup-totp
```

Comment:
**question:** Do you know if the next steps might take longer than 30 seconds until this token is used? 
If so, this step might need to run closer to the consumer step. Otherwise, the TOTP value might be expired when using it.

Reply:
Every command after `npm ci ` only took just a couple of 1-5 seconds.